main.go

```diff
@@ -80,6 +81,7 @@ type autographer struct {
 	debug         bool
 	heartbeatConf *heartbeatConfig
 	authBackend   authBackend
+	isReady       bool
```

Comment:
Might want to rename this to `lastHeartbeatOK` or at least document what it is / how it works